mu-plugins/central-hub/tests/phpunit/unit/config-store/_loadConfigFromFilesystem.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ *  Tests for _load_config_from_filesystem
+ *
+ * @package    spiralWebDb\centralHub\Tests\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\ConfigStore;
+
+use function KnowTheCode\ConfigStore\_load_config_from_filesystem;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_LoadConfigFromFilesystem
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_LoadConfigFromFilesystem extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/internals.php';
+	}
+
+	/**
+	 *  Test should return false when $path_to_file is empty string.
+	 */
+	public function test_should_return_false_when_path_to_file_is_empty_string() {
+		$expected = false;
+
+		$this->assertFalse( _load_config_from_filesystem( '') );
+		$this->assertSame( $expected, _load_config_from_filesystem( '' ) );
+	}
+
+	/**
+	 *  Test should return array key and current array when a configuration array is given.
+	 */
+	public function test_should_return_array_key_and_current_array_from_config() {
```

Comment:
What is this test doing?  It's not testing the `_load_config_from_filesystem` function.

Here, you want to test that the configuration array is returned when you specify a valid path/to/file.  Right?  

Here's what to do:

1. In `tests/phpunit/` create a folder called `fixtures`.
2. Add a dummy configuration file and then add the configuration parameters to it.
3. Then in this test:
      - give `_load_config_from_filesystem` the path/to/config file you just created in the `fixtures` folder
      - test that it sends back the array in the right format, i.e. as expected

Reply:
@hellofromtonya [1] A `fixtures` directory was added to `/tests/phpunit`; [2] a dummy configuration array was added; and [3] the 'path/to/file' was assigned to the variable `$path_to_file` within an updated test for the function `_load_config_from_filesystem`.  

A new test to evaluate that the array fixture was loaded was written and all assertions passed.